falcon/request.py

```diff
@@ -838,6 +838,43 @@ def get_param_as_list(self, name,
 
         raise HTTPMissingParam(name)
 
+    def get_param_as_date(self, name, format_string='%Y-%m-%d',
+                          required=False, store=None):
+        """
+        Args:
+            name (str): Parameter name, case-sensitive (e.g., 'ids').
+            format_string (str): String used to parse the param value into a date. # NOQA
```

Comment:
The NOQA actually gets rendered in the sphinx docs, so we'll have to come up with a different solution. For this line, it can simply be wrapped.